Dockerfile

```diff
@@ -3,7 +3,7 @@ ENV PYTHONUNBUFFERED 1
 RUN mkdir /code
 ADD . /code/
 WORKDIR /code
-RUN make reqs-install-dev
 RUN pip install --upgrade pip
 RUN pip install pip-tools
-RUN pip install psycopg2-binary
\ No newline at end of file
+RUN make reqs-install-dev
+RUN pip install debugpy
```

Comment:
@dtisza1 @JFU-GIT Any opinions on whether this should be vendored and installed as a part of requirements.dev.txt or not?  I don't have any opinions myself since I don't really use this functionality but checking for consistency sake.

Reply:
did not vendor it, since it's kind of one up use

Reply:
can see the use case of vendor the debugger for non docker use..., making change